9.2/root/usr/bin/run-postgresql

```diff
@@ -16,5 +16,10 @@ fi
 
 set_passwords
 
+if [ -f ${CONTAINER_SCRIPTS_PATH}/post-init.sh ]; then
+  log_info 'Sourcing post-init.sh ...'
+  source ${CONTAINER_SCRIPTS_PATH}/post-init.sh
```

Comment:
@pkajaba I share the question with @mnagy, why does this need to be a shell script and why use `source`?

I would expect any executable would do.